src/gui/MainWindow.cc

```diff
@@ -1103,6 +1103,8 @@ void MainWindow::compile(bool reload, bool forcedone)
     compileDone(didcompile | forcedone);
   } catch (const HardWarningException&) {
     exceptionCleanup();
+  } catch (std::exception &ex) {
```

Comment:
Should we go with `const` ref?
```suggestion
  } catch (const std::exception& ex) {
```

Reply:
Ah, yes, sure.